client/blocks/inline-help/inline-help-query-support-types.jsx

```diff
@@ -53,7 +51,6 @@ class QueryInlineHelpSupportTypes extends Component {
 		return (
 			<Fragment>
 				<QueryTicketSupportConfiguration />
-				<QuerySupportHistory email={ this.props.currentUserEmail } />
```

Comment:
One benefit of removing `QuerySupportHistory` from here is that the history is not loaded right when you open the help popup, but only after you select the "Contact us" tab.

As a followup, the entire `QueryInlineHelpSupportTypes` should be moved from the `InlineHelpPopover` component down to the `InlineHelpContactView` component which is the actual one that really needs to fetch all these data.

Reply:
Good call. I'll follow up with this in another PR.

Reply:
See #59116.